apps/ringtones/js/share.js

```diff
@@ -228,12 +228,9 @@ function handleShare(data, callback) {
         // Show a notification indicating success, and then close it immediately
         // so it doesn't stink up the notifications tray! XXX: This UX isn't
         // great; we should turn this into a transient notification when we can.
-        new Notification(
-          songtitle,
-          {body: navigator.mozL10n.get(
-            details.setAsDefault ? 'set-default-tone' : 'created-tone'
-          )}
-        ).close();
+        NotificationHelper.send(songtitle, {
```

Comment:
because it's a raw string, and not l10nId, you actually need to do:

NotificationHelper.send({raw: songtitle}, ...); see https://github.com/mozilla-b2g/gaia/blob/9946a490a9264b42e65385d703b28fa055ab2d42/shared/js/notification_helper.js#L30